src/secure_random.cr

```diff
@@ -92,7 +101,7 @@ module SecureRandom
     @@initialized = true
 
     {% if flag?(:linux) %}
-      if getrandom(Bytes.new(16)) >= 0
+      if sys_getrandom(Bytes.new(16)) >= 0
```

Comment:
This should still call `getrandom, even for 16 bytes. Maybe we could only read 2 bytes at first.